beacon_node/Cargo.toml

```diff
@@ -15,6 +15,9 @@ path = "src/lib.rs"
 write_ssz_files = [
     "beacon_chain/write_ssz_files",
 ] # Writes debugging .ssz files to /tmp during block processing.
+fast_confirmation = [
+    "fork_choice/fast_confirmation",
+] # Enable Fast Confirmation Rule (FCR) for faster block confirmation.
```

Comment:
I'm actually fine with not feature-gating FCR if it can be enabled and disabled at runtime. I think this will just slow down prototype development, and would be unnecessary for eventual integration into Lighthouse.